htdocs/core/modules/project/task/modules_task.php

```diff
@@ -58,7 +58,7 @@ public static function liste_modeles($db, $maxfilenamelength = 0)
 
 
 /**
- *  Classe mere des modeles de numerotation des references de projets
+ * Parent class of project reference numbering models
```

Comment:
Good try !
But this is the template for tasks not project.
Try again. A good exercice to resubmit a PR after other changes were merged ...  ;-)